resources/assets/js/helpers/localize.ts

```diff
@@ -13,7 +13,10 @@ export function localizeField<T>(
   model: T,
   field: TranslatableKeys<T>,
 ): string | null {
-  return model[field][locale];
+  if (model[field] && model[field][locale]) {
```

Comment:
I'd do an explicit `!== undefined` check, and not bother to check `model[field][locale]`. As it is, if `model[field][locale]` is the empty string it will evaluate as falsey and will return null instead.

Reply:
Added a `!== null` check, `undefined` was still failing and throwing an error.